docs/frameworks/additional-requests.rst

```diff
@@ -0,0 +1,228 @@
+.. _framework-additional-requests:
+
+==============================
+Supporting Additional Requests
+==============================
+
+To support :ref:`additional requests <advanced-requests>`, your framework must
+provide the request download implementation of :class:`~.HttpClient`.
```

Comment:
Could you please check

> The key words "MUST”, "MUST NOT”, "REQUIRED”, "SHALL”, "SHALL NOT”, "SHOULD”,
"SHOULD NOT”, "RECOMMENDED”, "MAY”, and "OPTIONAL” in this document are to be
interpreted as described in RFC `2119 <https://www.ietf.org/rfc/rfc2119.txt>`_.

in the page-pbjects/additional-requests? I think this section applies mostly to the framework docs, and if we keep it, we should keep it for framework docs only.

Reply:
I have removed it for now. Once we work on the framework docs, and have an introduction page for them, we *can* consider adding it back. Though I am not sure we need it.